<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RocketGroundComProtolClass</class>
 <widget class="QMainWindow" name="RocketGroundComProtolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RocketGroundComProtol</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="7" column="0">
       <widget class="QLabel" name="status">
        <property name="text">
         <string>运行状态:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="startSerialBtn">
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="statusDisplay">
        <property name="text">
         <string>停止.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="serialPort">
        <property name="text">
         <string>串口:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="5">
       <widget class="QPlainTextEdit" name="dataDisplay"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="waitTimeOut">
        <property name="text">
         <string>等待超时时间:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="timeOutSet"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="recieveData">
        <property name="text">
         <string>接收数据:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="serialPortNameCx"/>
      </item>
      <item row="7" column="3">
       <widget class="QLabel" name="trafficDisplay">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>响应:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RocketGroundComProtol.qrc"/>
 </resources>
 <connections/>
</ui>
